tests/test_kernels.py

```diff
@@ -644,3 +664,293 @@ def test_grad_mask_function():
     grad = [1, 2, 3, 4, 5]
     hyps_mask = {'map': [0, 3, 5]}
     assert (from_grad_to_mask(grad, hyps_mask) == [1, 4]).all()
+
+
+# -----------------------------------------------------------------------------
+#                              test many body kernels
+# -----------------------------------------------------------------------------
+
+
+def test_many_body_force():
+    """Check that the analytical force kernel matches finite difference of
```

Comment:
This chunk of code can be simplified via loops....

Reply:
this will be address in pull #166